app/forms/offences_form.rb

```diff
@@ -0,0 +1,11 @@
+class OffencesForm < Forms::Base
+  property :release_date,           type: TextDate,   validates: { presence: true }
+  property :not_for_release,        type: Axiom::Types::Boolean
+  property :not_for_release_reason, type: String
+
+  validate :validate_release_date
+
+  def validate_release_date
+    errors.add(:release_date) unless release_date.is_a? Date
+  end
+end
```

Comment:
In order to avoid these warnings on missing new line at the end of the file, maybe you need to set your editor to do that automatically.